python/test_infra/ttir_builder.py

```diff
@@ -304,9 +304,7 @@ def eltwise_proxy(
                 inputs,
                 [output],
                 self._get_operand_constraint_attr(3),
-                loc=Location.file(
-                    filename=stack[0].filename, line=stack[0].lineno, col=id
-                ),
+                loc=Location.name(str(id)),
```

Comment:
fyi I still need to change this back

Reply:
fyi @ctodTT 

@tapspatel can you file an issue to support the new way e2e and assign to Collin?

Reply:
will do